### DISCUSSION TITLE
Unicorn Mode C harness

### DISCUSSION THREAD

**Discussion by wardjm** [AFLplusplus/AFLplusplus#1190] [Q&A]
Has anyone written a C unicorn_loader for loading the context dir the way the Python unicorn_loader does it? This doesn't seem super difficult, but it would be a decent amount of work to redo if it has already been done.

**Comment by domenukk** [comment]
I would suggest doing this using the rust bindings fwiw, as it should be a lot less work.
Personally I use unicorn for emulation mostly, manually loading the necessary pieces into memory.
